packages/hardhat/contracts/FundingVaultFactory.sol

```diff
@@ -76,6 +76,7 @@ contract FundingVaultFactory{
      * @param _projectURL A link or hash containing the project's information (e.g., GitHub repository).
      */
     function deployFundingVault (
+		address _fundingToken,
```

Comment:
Indentation seems wrong here. It is probably an issue in your editor's configuration about how to interpret tabs.